raft.go

```diff
@@ -1644,11 +1646,15 @@ func stepFollower(r *raft, m pb.Message) error {
 }
 
 func (r *raft) handleAppendEntries(m pb.Message) {
+	entries, err := VerifyLogRange(m.Index, m.LogTerm, m.Entries)
```

Comment:
It may cause reduce of performance. So suggest to only enable the verification in test by default; and disable it in production. 

We can introduce an environment variable to enable / disable it.  FYI. https://github.com/etcd-io/etcd/blob/main/client/pkg/verify/verify.go